packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -1201,7 +1201,7 @@ contract SphinxUtils is SphinxConstants, StdUtils {
      */
     function makeGnosisSafeTransaction(
         Vm.AccountAccess memory _access
-    ) private pure returns (GnosisSafeTransaction memory) {
```

Comment:
Do you think we should have an integration test for this feature? Your call. Just floating the idea

Reply:
Yeah, I thought about that. Since this functionality takes advantage of our existing systems for generating execute actions from AccountAccesses and because we already have a forge test to confirm this action will revert on-chain, I don't think an integration test would add any coverage.